REFERENCE.md

```diff
@@ -66,12 +67,17 @@ The following parameters are available in the `profile_monitoring::telegraf` cla
 * [`config_dirs_default_group`](#config_dirs_default_group)
 * [`config_dirs_default_mode`](#config_dirs_default_mode)
 * [`config_dirs_default_owner`](#config_dirs_default_owner)
+* [`group`](#group)
+* [`homedir`](#homedir)
 * [`inputs_extra`](#inputs_extra)
 * [`inputs_extra_scripts`](#inputs_extra_scripts)
 * [`ipmi_sensor_plugin_enabled`](#ipmi_sensor_plugin_enabled)
 * [`ipmi_sensor_telegraf_plugin_options`](#ipmi_sensor_telegraf_plugin_options)
 * [`outputs`](#outputs)
 * [`required_pkgs`](#required_pkgs)
+* [`uid`](#uid)
+* [`user`](#user)
+* [`user_comment`](#user_comment)
```

Comment:
Apparently my last commit/PR to this repo forgot to include updates to the REFERENCE file. These changes are related to managing the UID of telegraf, rather than this PR. Sorry.